examples/AT start up.ipynb

```diff
@@ -0,0 +1,422 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook contains a template for basic startup operations for the Auxiliary telescope. At the begging of the night a user can run the startup procedure to prepare the telescope for observing. Omce it is done, remember to shutdown the notebook server. \n",
+    "\n",
+    "We start by loading all the libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "import yaml\n",
+    "\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "import astropy.units as u\n",
+    "from astropy.time import Time\n",
+    "from astropy.coordinates import AltAz, ICRS, EarthLocation, Angle, FK5\n",
+    "import asyncio\n",
+    "\n",
+    "# import palpy\n",
+    "\n",
+    "from lsst.ts import salobj\n",
+    "\n",
+    "from lsst.ts.standardscripts.auxtel.attcs import ATTCS\n",
+    "\n",
+    "from lsst.ts.idl.enums import ATPtg\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.utils import iers\n",
+    "iers.conf.auto_download = False"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setup remotes for all the AT components"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "%%script bash \n",
```

Comment:
Maybe just explain when it is required to do this. Maybe put it in a separate cell and say, "uncomment and run if IDL files don't exist" or something like that.